pkg/entity/batch_exporter/batch_exporter.go

```diff
@@ -69,13 +70,21 @@ func create(streamName *string, cmd *cobra.Command) {
 
 	pathPrefix := util.GetStringAndErr(flags, pathPrefix)
 
+	projectName := util.GetStringAndErr(flags, projectName)
```

Comment:
Two things here:
1. the flag name `projectName` is reassigned to the value here, which is confusing. Better to rename the `projectName` flag to `projectNameFlag` as is done with the other flags as well.
2. Maybe we should create a global flag for `project`, as that is used pretty much everywhere. We could then just ignore the flag whenever it is inappropriate. Also, this if-else statement can then be generalized somewhere in the codebase to be reused.

Reply:
1. agreed
2. It's only used in the create commands, if we want to extend that, that's fine. For now I disagree with a global flag